src/txn/txn_ckpt.c

```diff
@@ -319,10 +319,12 @@ __wt_checkpoint_get_handles(WT_SESSION_IMPL *session, const char *cfg[])
 			 * safely be skipped here.
 			 */
 			F_CLR(&session->txn, WT_TXN_ERROR);
-			if (force)
-				WT_RET_MSG(session, EBUSY,
+			if (force) {
+				WT_RET(__wt_msg(session,
```

Comment:
I chose to propagate the error from `__wt_msg` rather than EBUSY, since errors from `__wt_msg` seem more concerning. I could be convinced to go the other way